### ISSUE TITLE
SGD but no learning rate decay found

### ISSUE THREAD

**Issue by zhuotest** [chenyuntc/simple-faster-rcnn-pytorch#51]
It is hard coded in `train.py` that when reached 9 epoch, do learning rate decay.

This is not elegant. Also, waste people's time if they didn't notice this setting.

Please, set this hard coded learing rate stepsize in config.py
